src/backend/web/graphql/index.js

```diff
@@ -1,26 +1,23 @@
 const { gql: graphql } = require('apollo-server-express');
 
-const {
-  getFeedsCount,
-  getFeed,
-  getFeeds,
-  getPostsCount,
-  getPosts,
-} = require('../../utils/storage');
+const { getFeedsCount, getFeeds, getPostsCount, getPosts } = require('../../utils/storage');
 
-const Post = require('../../post');
+const Post = require('../../data/post');
+const Feed = require('../../data/feed');
 
 const maxPostsPerPage = process.env.MAX_POSTS_PER_PAGE || 30;
 
 module.exports.typeDefs = graphql`
   # 'Feed' matches our Feed type used with redis
   type Feed {
+    id: String
     name: String
```

Comment:
We have author in our `Feed` class, we could use that here for consistency.

Reply:
Good idea, I'll do that.